src/aws-api-mcp-server/awslabs/aws_api_mcp_server/server.py

```diff
@@ -219,6 +220,26 @@ async def call_aws(
     ] = None,
 ) -> ProgramInterpretationResponse | AwsApiMcpServerErrorResponse | AwsCliAliasResponse:
     """Call AWS with the given CLI command and return the result as a dictionary."""
+    return await call_aws_helper(
+        cli_command=cli_command,
+        ctx=ctx,
+        max_results=max_results,
+        credentials=None,
+    )
+
+
+async def call_aws_helper(
```

Comment:
Have we considered if this needs to be MCP aware? I imagine some callers might want to avoid the MCP overhead and call this method directly. 

Besides this, we should add more nuance to the description especially if this helper is a public capability.